### PR TITLE
Design 01

### PR DESCRIPTION
**Pull request by leakhairallah**

### CODE REVIEW

**Review by dorsaffrigui — COMMENTED** [review]
Could you add a .env.template file to your project and document that in the Readme.
You can find here an [example](https://github.com/Finastra/angular-nestjs-starter#environement-variables)

**Review by dorsaffrigui — COMMENTED** [review]
Could you provide the different commands (to build the client and serve the api) in both npm and yarn

**Review by dorsaffrigui — COMMENTED** [review]
Use the[ fds-divider](https://finastra.github.io/finastra-design-system/?path=/story/data-display-divider--default-story) between to separate the transactions

Screenshot 2022-08-11 at 11 22 21

**Review by dorsaffrigui — COMMENTED** [review]
You can change the Dialog width to make it bigger using the --fds-dialog-min-width css var
Screenshot 2022-08-11 at 11 19 33

**Review by dorsaffrigui — COMMENTED** [review]
I got this error when I tried to make a money transfer 

Screenshot 2022-08-11 at 11 20 08

**Review by dorsaffrigui — COMMENTED** [review]
The account card does't look like the design system cards, you can use the[ fds-account-card](https://finastra.github.io/finastra-design-system/?path=/story/data-display-cards-account-card--default)

Screenshot 2022-08-11 at 11 45 38

**Review by dorsaffrigui — COMMENTED** [review]
Could you move the different components that you have under the retail-app/src into libs/components for example and try to have only the retails/src/app.

Thus, it will be easier to find the files to modify.

File: apps/retail-app/src/app/app.js [hunk 1]
```diff
@@ -3,25 +3,70 @@ import './app.scss';
...
 import '@finastra/user-profile';
+import '@finastra/dialog';
 import '@material/mwc-icon-button';
 import '@finastra/sidenav';
 import '@finastra/logo';
 import '@material/mwc-list';
 import '@finastra/logo';
-import { Grid } from '@mui/material';
+import '@finastra/checkbox';
+import { Grid, Box, Paper } from '@mui/material';
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.js:13]:
```suggestion
import { Grid } from '@mui/material';
```

File: apps/retail-app/src/account-component/account-component.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+html,
+body {
+    &::-webkit-scrollbar {
+        display: none;
+    }
+}
+
+@media (prefers-color-scheme: dark) {
+  body {
+    @include fds.use-dark-theme();
```
Comment (dorsaffrigui) [inline apps/retail-app/src/account-component/account-component.scss:81]: You don't have to include the dark theme in every component. 
it should be enough to do it only from your app.scss or styles.scss

File: apps/retail-app/src/app/app.scss [hunk 3]
```diff
@@ -1,65 +1,101 @@
...
         padding: var(--fds-spacing-0) var(--fds-spacing-3) var(--fds-spacing-2) var(--fds-spacing-3);
       }
+
       fds-user-profile fds-button + fds-button {
         margin-top: var(--fds-spacing-2);
       }
+
       .fds-sidenav {
         --mdc-drawer-width: 300px;
         background-color: var(--fds-surface);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:24]:
```suggestion
        background-color: var(--fds-surface);
        @include fds.property(background-color,  surface);
```

File: apps/retail-app/src/app/app.scss [hunk 4]
```diff
@@ -1,65 +1,101 @@
...
         color: var(--fds-on-surface);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:25]:
```suggestion
        color: var(--fds-on-surface);
        @include fds.text-color(on-surface);
```

File: apps/retail-app/src/app/app.scss [hunk 5]
```diff
@@ -1,65 +1,101 @@
...
       }
+
       .fds-sidenav-header {
         background-color: var(--fds-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:29]:
```suggestion
        @include fds.property(background-color,  background);
```

File: apps/retail-app/src/app/app.scss [hunk 6]
```diff
@@ -1,65 +1,101 @@
...
       }
 
       .header {
-        width:100%;
-        height:180px;
-        background-color: var(--fds-primary); 
+        background: url("../assets/Background.png") no-repeat;
+        height: 350px;
+        background-size: cover;
+        background-color: var(--fds-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:45]:
```suggestion
        @include fds.property(background-color,  background);
```

File: apps/retail-app/src/app/app.scss [hunk 7]
```diff
@@ -1,65 +1,101 @@
...
+
+      .app-title{
+        float: left;
+        margin-left: 10px;
+        font: var(--fds-headline-3);
+        color: var(--fds-surface);
+      }
+
+      .welcome-msg{
+        font: var(--fds-body-1);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:68]:
```suggestion
        @include fds.typography(body-1);
```

File: apps/retail-app/src/app/app.scss [hunk 8]
```diff
@@ -1,65 +1,101 @@
...
+        color: var(--fds-surface);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:69]:
```suggestion
        @include fds.text-color(surface);
```

File: apps/retail-app/src/app/app.scss [hunk 9]
```diff
@@ -1,65 +1,101 @@
...
+      }
+
+      .content{
+        padding-top: 48px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:73]:
```suggestion
         @include fds.padding(5, top);
```

File: apps/retail-app/src/app/app.scss [hunk 10]
```diff
@@ -1,65 +1,101 @@
...
 
       .main-content{
-        padding-left: 10%;
+        display: flex;
+        justify-content: center;
         padding-right: 10%;
-        padding-top: 10%;
+        padding-left: 10%;
+        padding-top: 5%;
+        background-color: var(--fds-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:82]:
```suggestion
         @include fds.property(background-color,  background);
```

File: apps/retail-app/src/carousel-component/carousel-component.scss [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+        padding: 12px;
+        padding-right: 24px;
+    }
+    
+    .left-arrow, .right-arrow {
+        float: right;
+    }
+    
+    .button-icon{
+        color: var(--fds-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/carousel-component/carousel-component.scss:72]:
```suggestion
        @include fds.text-color(background);
```

File: apps/retail-app/src/carousel-component/carousel-component.scss [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+html,
+body {
+    &::-webkit-scrollbar {
+        display: none;
+    }
+}
+
+@media (prefers-color-scheme: dark) {
+  body {
+    @include fds.use-dark-theme();
```
Comment (dorsaffrigui) [inline apps/retail-app/src/carousel-component/carousel-component.scss:85]: Remove all the includes in every component

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 13]
```diff
@@ -0,0 +1,42 @@
+@use '@finastra/fds-theme' as fds;
+
+body {
+  @include fds.use-light-theme();
+  .chart{
+      width: 100%;
+      height: 100%;
+      border-radius: 4px;
+      background-color: var(--fds-surface);;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:9]:
```suggestion
      @include fds.property(background-color,  surface);
```

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+      box-shadow: rgba(50, 50, 105, 0.15) 0px 2px 5px 0px, rgba(0, 0, 0, 0.05) 0px 1px 1px 0px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:10]:
```suggestion
       @include fds.elevation(4);
```

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+      overflow: hidden;
+  }
+
+  .chart-content{
+    padding: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:15]:
```suggestion
    @include fds.padding(4);
```

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  .donut-chart{
+    display: flex;
+    justify-content: center;
+    padding-bottom: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:21]:
```suggestion
    padding-bottom: 24px;
    @include fds.padding(4, bottom);
```

File: apps/retail-app/src/app/app.scss (hunk 10 above)
Comment (dorsaffrigui) [inline apps/retail-app/src/app/app.scss:82]:
```suggestion
         @include fds.property(background-color,  background);
```

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  .chart-title{
+    font: var(--fds-headline-5);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:25]:
```suggestion
    @include fds.typography(headline-5);
```

File: apps/retail-app/src/chart-component/chart-component.scss [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+    color: var(--fds-on-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/chart-component/chart-component.scss:26]:
```suggestion
    @include fds.text-color(on-background);
```

File: apps/retail-app/src/myadvisor-component/myadvisor-widget.scss [hunk 19]
```diff
@@ -17,6 +21,7 @@ body {
         border-bottom: 1px solid;
         border-color: rgb(217, 218, 217);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/myadvisor-component/myadvisor-widget.scss:22]:
```suggestion
        @include fds.property(border-color, outline)
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 20]
```diff
@@ -1,42 +1,80 @@
 @use '@finastra/fds-theme' as fds;
 
 body {
+    @include fds.use-light-theme();
     .quick-transfer{
         width: 100%;
-        height: 328px;;
+        height: 100%;
         border-radius: 4px;
         background-color: var(--fds-surface);;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:9]:
```suggestion
        @include fds.property(background-color, surface);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 21]
```diff
@@ -1,42 +1,80 @@
...
         box-shadow: rgba(50, 50, 105, 0.15) 0px 2px 5px 0px, rgba(0, 0, 0, 0.05) 0px 1px 1px 0px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:10]:
```suggestion
      @include fds.elevation(4);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 22]
```diff
@@ -1,42 +1,80 @@
...
     }
 
     .quick-transfer-content{
         padding: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:14]:
```suggestion
       @include fds.padding(4);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 23]
```diff
@@ -1,42 +1,80 @@
...
+        overflow: auto;
     }
 
     .quick-transfer-title{
         font: var(--fds-headline-5);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:19]:
```suggestion
        @include fds.typography(headline-5);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 24]
```diff
@@ -1,42 +1,80 @@
...
         color: var(--fds-on-background);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:20]:
```suggestion
        @include fds.text-color(on-background);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 25]
```diff
@@ -1,42 +1,80 @@
...
-        padding-top: 36px;
+    .quick-transer-inputs{
+        width: 100%;
+        height: 100%;
+        position: relative;
     }
 
-    .amount{
+    .sendto{
         padding-top: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:30]:
```suggestion
        @include fds.padding(4, top, bottom);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 26]
```diff
@@ -1,42 +1,80 @@
...
+        padding-bottom: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:31]:
```suggestion
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 27]
```diff
@@ -1,42 +1,80 @@
...
         width: 100%;
         height: 100%;
+        position: relative;
     }
 
-    .send-button{
+    .button{
         float: right;
-        padding: 24px;
+        padding-top: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:45]:
```suggestion
        @include fds.padding(4, top);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 28]
```diff
@@ -1,42 +1,80 @@
...
+        overflow: auto;
+    }
+
+    ::-webkit-scrollbar {
+        width: 10px;
+        height: 10px;
+    }
+    
+    ::-webkit-scrollbar-thumb {
+        background: var(--fds-surface-selected);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:59]:
```suggestion
        @include fds.bg-color(surface-selected);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 29]
```diff
@@ -1,42 +1,80 @@
...
+        border-radius: 4px;
+    }
+    
+    ::-webkit-scrollbar-track {
+        background: var(--fds-surface);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:64]:
```suggestion
        @include fds.bg-color(surface);
```

File: apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss [hunk 30]
```diff
@@ -1,42 +1,80 @@
...
+    }
+
+    fds-select{
+        padding-bottom: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/quicktransfer-component/quick-transfer-widget.scss:68]:
```suggestion
       @include fds.padding(4, bottom);
```

File: apps/retail-app/src/transaction-component/transaction-widget.scss [hunk 31]
```diff
@@ -12,6 +13,7 @@ body {
 
     .sheet-item{
         padding: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction-widget.scss:15]:
```suggestion
        @include fds.padding(4);
```

File: apps/retail-app/src/transaction-component/transaction-widget.scss [hunk 32]
```diff
@@ -24,21 +26,29 @@ body {
     }
     
     .layout{
-        width: 100%;
-        height: 100%;
         padding-top: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction-widget.scss:29]:
```suggestion
        @include fds.padding(4, top);
```

File: apps/retail-app/src/transaction-component/transaction-widget.scss [hunk 33]
```diff
@@ -24,21 +26,29 @@ body {
...
     }
 
     .transaction-list{
         padding-top: 24px;
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction-widget.scss:33]:
```suggestion
        padding-top: 24px;
        @include fds.padding(4, top);
```

File: apps/retail-app/src/transaction-component/transaction-widget.scss [hunk 34]
```diff
@@ -47,9 +57,31 @@ body {
...
+    }
+
+    ::-webkit-scrollbar {
+        width: 10px;
+        height: 10px;
+        
+    }
+    
+    ::-webkit-scrollbar-thumb {
+        background: var(--fds-surface-selected);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction-widget.scss:72]:
```suggestion
        @include fds.bg-color(surface-selected);
```

File: apps/retail-app/src/transaction-component/transaction-widget.scss [hunk 35]
```diff
@@ -47,9 +57,31 @@ body {
...
+        border-radius: 4px;
+    }
+    
+    ::-webkit-scrollbar-track {
+        background: var(--fds-surface);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction-widget.scss:77]:
```suggestion
        @include fds.bg-color(surface);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 36]
```diff
@@ -16,8 +16,15 @@ body {
         border-bottom: 1px solid;
         border-color: rgb(217, 218, 217);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:17]:
```suggestion
       @include fds.property(border-color, outline);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 37]
```diff
@@ -16,8 +16,15 @@ body {
...
     }
+
+    .transaction-last{
+        border-color: rgb(217, 218, 217);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:21]:
```suggestion
        @include fds.property(border-color, outline);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 38]
```diff
@@ -28,22 +35,23 @@ body {
     .transaction-name{
         width: 90%;
         height: 100%;
-        padding-left: 50px;
+        padding-left: 12px;
     }
 
     .transaction-title{
         font: var(--fds-subtitle-2);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:42]:
```suggestion
        @include fds.typography(subtitle-2);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 39]
```diff
@@ -28,22 +35,23 @@ body {
...
+        display: flex;
+        justify-content: space-between;
     }
 
     .transaction-subtitle{
         font: var(--fds-body-3);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:48]:
```suggestion
        @include fds.typography(body-3);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 40]
```diff
@@ -58,3 +66,24 @@ body {
     @include fds.use-dark-theme();
   }
 }
+
+
+@media only screen and (max-width: 415px) {
+    body{
+        .transaction-title{
+            font: var(--fds-subtitle-3);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:74]:
```suggestion
            @include fds.typography(subtitle-3);
```

File: apps/retail-app/src/transaction-component/transaction.scss [hunk 41]
```diff
@@ -58,3 +66,24 @@ body {
...
+            display: flex;
+            justify-content: space-between;
+        }
+
+        .transaction-subtitle{
+            font: var(--fds-body-3);
```
Comment (dorsaffrigui) [inline apps/retail-app/src/transaction-component/transaction.scss:80]:
```suggestion
            @include fds.typography(body-3);
```

**Comment by leakhairallah** [comment]
> 



> The account card does't look like the design system cards, you can use the[ fds-account-card](https://finastra.github.io/finastra-design-system/?path=/story/data-display-cards-account-card--default)
> 
> Screenshot 2022-08-11 at 11 45 38

Resolved

**Comment by leakhairallah** [comment]
> I got this error when I tried to make a money transfer
> 
> Screenshot 2022-08-11 at 11 20 08

Resvoled

**Pull request merged**